readme.md

```diff
@@ -411,6 +412,18 @@ have the following in your `:CocConfig`.
 ```json
 "coc.preferences.formatOnSaveFiletypes": ["scala"]
 ```
+### Analyzing Stacktraces
+
+![anaylyz-stacktrace](https://i.imgur.com/tQjiAG2.gif)
+
+To utilize the [analyze
+stacktrace](https://scalameta.org/metals/docs/editors/new-editor.html#analyze-stacktrace)
+command you'll want to copy your your stacktrace to your register. Once copied,
+simply execute the `metals.analyze-stacktrace` command. This should create a
+`.metals/stacktrace.scala` file which will have code lenses which will allow you
+to jump to the part of your codebase listed in the stacktrace. Keep in mind that
+in order to utlize this feature you'll want to make sure you have
+`"codeLens.enable": true` in your config.
```

Comment:
code lenses for stacktrace will be provided regardless of this setting(same as run/test lenses)

Reply:
Well this setting is actually a bit different. It's for coc.nvim and it actually completely turns off the code lens settings, so the requests will never be sent to the server, an if by chance they are sent, they won't be handled. It exists because by default it's off since Vim doesn't have virtual text, only Nvim. So code lenses aren't handled by default. Without this, the lenses won't show.

Also, this reminded me that I need to make a note that this will only work in Nvim.